extensions/github/src/auth.ts

```diff
@@ -57,25 +59,30 @@ export function getOctokit(): Promise<Octokit> {
 
 let _octokitGraphql: Promise<graphql> | undefined;
 
-export function getOctokitGraphql(): Promise<graphql> {
+export async function getOctokitGraphql(): Promise<graphql> {
 	if (!_octokitGraphql) {
-		_octokitGraphql = getSession()
-			.then(async session => {
-				const token = session.accessToken;
-				const { graphql } = await import('@octokit/graphql');
-
-				return graphql.defaults({
-					headers: {
-						authorization: `token ${token}`
-					},
-					request: {
-						agent: getAgent()
-					}
-				});
-			}).then(null, async err => {
-				_octokitGraphql = undefined;
-				throw err;
+		try {
+			const session = await authentication.getSession('github', scopes, { createIfNone: false });
```

Comment:
```suggestion
			const session = await authentication.getSession('github', scopes);
```
`false` is the default.